Python/jit.c

```diff
@@ -390,66 +406,128 @@ patch_x86_64_32rx(unsigned char *location, uint64_t value)
     patch_32r(location, value);
 }
 
+void patch_aarch64_trampoline(unsigned char *location, int ordinal, CompileState *state);
```

Comment:
If you move the function def up here, no declaration is necessary.

Reply:
Cannot move it because the function uses `symbols_map` which is defined in jit_stencils.h.
In jit_stencils.h also references this function hence it needs a declaration here.

Reply:
I think we can break this circular reference if we create a new .h file.

Reply:
It's fine, I missed the circular dependency.